tvb_build/tvb_build/third_party_licenses/packages_accepted.xml

```diff
@@ -3,14 +3,226 @@
+        <copyright-notice value="Copyright 2009-2024 Michael Bayer."/>
         <description
                 value=" Alembic is a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python"/>
     </dependency>
+    <dependency env="mac" name="accessibility">
+        <full-name value="accessibility"/>
+        <project-home value="https://github.com/ronaldoussoren/pyobjc/tree/master/pyobjc-framework-Accessibility"/>
+        <version value="unknown"/>
+        <usage value="dynamic linking"/>
+        <license value="LICENSE_pyobjc.txt"/>
+        <license-type value="MIT_"/>
+        <copyright-notice value="Copyright 2009-2024 Michael Bayer."/>
+        <description
+                value=" Alembic is a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python"/>
+    </dependency>
+    <dependency env="mac" name="accessibility">
+        <full-name value="Accessibility"/>
+        <project-home value="https://github.com/atheriel/accessibility"/>
+        <version value="unknown"/>
+        <usage value="dynamic linking"/>
+        <license value="LICENSE_accessibility.txt"/>
+        <license-type value="ISC_"/>
+        <copyright-notice value="Copyright 2013-2014, Aaron Jacobs (atheriel@gmail.com)"/>
+        <description value="It can be used to query and modify attributes of running applications, as well as watch for a variety of notifications."/>
+    </dependency>
+    <dependency env="mac" name="accessibility">
+        <full-name value="Accessibility"/>
+        <project-home value="http://pythonhosted.org/pyobjc/"/>
+        <version value="unknown"/>
+        <usage value="dynamic linking"/>
+        <license value="LICENSE_PyObjC.txt"/>
+        <license-type value="MIT_"/>
+        <copyright-notice value="Copyright (c) 2002, 2003 - Bill Bumgarner, Ronald Oussoren, Steve Majewski, Lele Gaifax, et.al., 2003-2024 - Ronald Oussoren"/>
+        <description value="We use this library for the Mac OS TVB distribution to appear in the doc bar."/>
+    </dependency>
+    <dependency env="mac" name="accounts">
+        <full-name value="Accounts"/>
+        <project-home value="http://pythonhosted.org/pyobjc/"/>
+        <version value="unknown"/>
+        <usage value="dynamic linking"/>
+        <license value="LICENSE_PyObjC.txt"/>
+        <license-type value="MIT_"/>
+            <copyright-notice value="Copyright (c) 2002, 2003 - Bill Bumgarner, Ronald Oussoren, Steve Majewski, Lele Gaifax, et.al., 2003-2024 - Ronald Oussoren"/>
+        <description value="We use this library for the Mac OS TVB distribution to appear in the doc bar."/>
+    </dependency>
+    <dependency env="mac" name="addressbook">
+        <full-name value="Addressbook"/>
+        <project-home value="http://pythonhosted.org/pyobjc/"/>
+        <version value="unknown"/>
+        <usage value="dynamic linking"/>
+        <license value="LICENSE_PyObjC.txt"/>
```

Comment:
All the packages with the license LICENSE_PyObjC.txt are from the same repo: https://github.com/ronaldoussoren/pyobjc. However, I will update project-home value since the correct one is https://github.com/ronaldoussoren/pyobjc

Reply:
maybe write this in the `description ` ?